tools/bbcert/bbcert.go

```diff
@@ -0,0 +1,80 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"os"
+	"os/exec"
+	"path"
+
+	"github.com/buildbuddy-io/buildbuddy/server/util/flag"
+	"github.com/buildbuddy-io/buildbuddy/server/util/grpc_client"
+	"github.com/buildbuddy-io/buildbuddy/server/util/log"
+
+	cgpb "github.com/buildbuddy-io/buildbuddy/proto/certgenerator"
+)
+
+var (
+	server = flag.String("server", "", "gRPC target for the certificate server")
+)
+
+const (
+	sshDir = ".ssh"
+	// For now, assume user is using an RSA key.
+	defaultRSAPublicKeyFile   = "id_rsa.pub"
+	defaultRSACertificateFile = "id_rsa-cert.pub"
```

Comment:
might be nice to put something in this filename that indicates it's ours. Would help folks distinguish this key from other keys they might have

Reply:
The certificate is tied to the user's key pair and it has to be named <key>-cert.pub for ssh to be able to find it. 
For convenience, I'm using the key that the user should already have on their system.

If we want to name it something else, we'd also have to have the tool generate a separate key pair. 
We could do that but I don't see a real benefit in doing so.